lib/components/T3Frame/T3Frame.js

```diff
@@ -2,7 +2,6 @@
 // All props are returned from appearance
 export default {
   name: 'T3Frame',
-  functional: true,
```

Comment:
Thank you for adding id attribute, but why we have removed functional definition ? This component has no internal state, so from perfomance perspective is better to use functional component here. 
If you have another opinion please let me know

Reply:
if t3-frame is set to functional, t3-dynamic cannot assign an id to the t3-frame.
I wanted to keep the change clean & simple.

But I can rewrite it again?

Reply:
T3Frame is still able to inherit attributes from T3Renderer, but you have to add attrs field and use ctx.data.attrs

example
```js
        class: [...],
        attrs: ctx.data.attrs
```